### DISCUSSION TITLE
Is it safe to have a contact email?

### DISCUSSION THREAD

**Discussion by SuperGames-D** [community/community#55537] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've created a contact email and I would like to put it on my website.
I'm not using this email for anything else.

Is it safe to have it on my website? Can other people get personal information from it?

**Comment by SoundaryaKoutharapu** [comment, marked as answer]
That's a great way to make it easy for visitors to your website to get in touch with you!
This will help keep your personal and work emails separate.

Here are some steps you can take to ensure the safety and privacy of your contact email:

1. Use a CAPTCHA: To prevent spam and bots from sending you unwanted messages, you can add a CAPTCHA (Completely Automated Public Turing test to tell Computers and Humans Apart) to your contact form. This will require the user to complete a simple test to prove they're human before they can submit the form.

2.  Don't include personal information in your email signature: When you set up your contact email, avoid including any personal information (such as your phone number or home address) in your email signature. This will help protect your privacy.

By taking these steps, you can help ensure that your contact email is safe and secure for visitors to your website to use.

Hope this helps you out!

**Comment by ranikumarich9821214-source** [comment]
R
[Uploading google-starttls-domains.csv…]()

**Comment by ranikumarich9821214-source** [comment]
[contacts (1).csv](https://github.com/user-attachments/files/22225358/contacts.1.csv)
[google-starttls-domains.csv](https://github.com/user-attachments/files/22225359/google-starttls-domains.csv)

**Comment by krillmango** [comment]
Depends on the design of your email host's antispam system.   DCC is "best"

Distributed Checksum Clearinghouses
